QtCollider/primitives/prim_misc.cpp

```diff
@@ -41,9 +41,13 @@
 #include <QFontMetrics>
 #include <QDesktopWidget>
 #include <QStyleFactory>
-#include <QWebEngineSettings>
 #include <QCursor>
 
+#ifdef QT_WEBENGINE
+  #include "../widgets/QcWebView.h"
+  #include <QWebEngineSettings>
```

Comment:
I noticed there's no #ifdef'd code within this file aside from here; is this include even needed?